example/package.json

```diff
@@ -4,15 +4,18 @@
   "main": "index.js",
   "scripts": {
     "start": "expo start -c",
-    "android": "expo run:android",
-    "ios": "expo run:ios"
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web"
   },
   "dependencies": {
+    "@expo/metro-runtime": "~3.2.1",
     "@expo/vector-icons": "^14.0.0",
     "@gorhom/bottom-sheet": "^4",
-    "@react-native-community/slider": "4.5.2",
+    "@react-native-community/slider": "4.4.2",
```

Comment:
Downgraded this version to resolve https://github.com/callstack/react-native-slider/issues/617 which I could replicate while testing the sliders in the example app.